include/liblightnvm_cli.h

```diff
@@ -145,6 +146,7 @@ struct nvm_cli_opts {
 	char *file_output;	///< For NVM_CLI_OPT_FILE_OUTPUT
 	size_t dec_val;		///< For NVM_CLI_OPT_VAL_DEC
 	size_t hex_val;		///< For NVM_CLI_OPT_VAL_HEX
+    int terse;
```

Comment:
Maybe just move this one up beneath `verbose`, so it is hanging out with the familiar ones. And add the `NVM_CLI_OPT_TERSE` comment as with the other options.

Reply:
btw. double-check that the indentation matches the other attributes of the struct.

Reply:
Moved it up. Fixed indentation,